server/server.go

```diff
@@ -94,8 +94,8 @@ func WriteEnrichedResponse(w http.ResponseWriter, title, resp, color, style stri
 	w.Write(GenerateEnrichedSlashResponse(title, resp, color, style))
 }
 
-func PostExtraMessages(responseURL string, payload []byte) error {
-	req, err := http.NewRequest("POST", responseURL, bytes.NewBuffer(payload))
+func PostExtraMessages(ResponseURL string, payload []byte) error {
+	req, err := http.NewRequest("POST", ResponseURL, bytes.NewBuffer(payload))
```

Comment:
Usually variables are lowercased. Why not in this case?

Reply:
this was golanglint suggestion, that's I renamed it.

Reply:
Yes please